### PR TITLE
feat(relay): Push projectconfigs into Redis

### PR DESCRIPTION
**Pull request by untitaker**
Synchronize projectconfigs into Redis to make internal Relays faster.

This PR is incomplete even if successfully deployed, as we still need to hook some things in getsentry to make quotas behave correctly.

### CODE REVIEW

**Review by jan-auer — APPROVED** [review]
LGTM. @tonyo can you please double-check what's needed to add the new task queue?

**Comment by tonyo** [comment]
The queue should be created automatically, so 👍

File: src/sentry/tasks/relay.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    # but valid cache.
+    #
+    # If this was running at the end of the task, it would be more effective
+    # against bursts of updates, but introduces a different race where an
+    # outdated cache may be used.
+    debounce_key = _get_schedule_debounce_key(project_id, organization_id)
+    cache.delete(debounce_key)
+
+    if project_id:
+        projects = Project.objects.filter(id=project_id)
```
Comment (JTCunning) [inline src/sentry/tasks/relay.py]: Please have this call and the one on L45 be cached lookups.

**Review by JTCunning — DISMISSED** [review]

File: src/sentry/tasks/relay.py (hunk 1 above)
Comment (untitaker) [reply src/sentry/tasks/relay.py]: ✅

**Comment by untitaker** [comment]
Follow-ups:

https://github.com/getsentry/sentry/pull/16464
https://github.com/getsentry/sentry/pull/16487
https://github.com/getsentry/relay/pull/406

**Pull request merged**